manage.py

```diff
@@ -22,15 +22,10 @@
 app = create_app(os.environ.get("APP_CONFIG_FILE") or "development")
 manager = Manager(app)
 
-TEST_CMD = "py.test --cov-config .coveragerc --cov sourcemash \
+TEST_CMD = "py.test --cov-report term-missing --cov-config .coveragerc --cov . \
```

Comment:
worker_tasks isn't in the sourcemash folder. So I broadened the scope of pytest, but specified what to omit in the coveragerc (as you saw above).

term-missing will print out both the percentages and the missing lines so you don't need to run coverage report each time after running tests.